src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -181,6 +181,7 @@ const handleVisibleNameEditModal = () => {
 };
 const handleNameUpdate = (name: string) => {
     dashboardDetailState.name = name;
+    dashboardDetailOriginState.dashboardInfo.name = name;
```

Comment:
suggestion: Oh, it's not good idea to change origin state. 
1) get dashboard api when update name
2) or, use `dashboardDetailState.name` in name space.